pkg/cluster/sync.go

```diff
@@ -556,20 +556,13 @@ func (c *Cluster) syncRoles() (err error) {
 
 	// create list of database roles to query
 	for _, u := range c.pgUsers {
-		userNames = append(userNames, u.Name)
-		// add team member role name with rename suffix in case we need to rename it back
-		if u.Origin == spec.RoleOriginTeamsAPI && c.OpConfig.EnableTeamMemberDeprecation {
-			deletedUsers[u.Name+c.OpConfig.RoleDeletionSuffix] = u.Name
-			userNames = append(userNames, u.Name+c.OpConfig.RoleDeletionSuffix)
-		}
-	}
-
-	// add team members that exist only in cache
-	// to trigger a rename of the role in ProduceSyncRequests
-	for _, cachedUser := range c.pgUsersCache {
-		if _, exists := c.pgUsers[cachedUser.Name]; !exists {
-			userNames = append(userNames, cachedUser.Name)
+		pg_role := u.Name
```

Comment:
`pg_role` should be `pgRole` because of the golang [naming conventions](https://golang.org/doc/effective_go#mixed-caps).